flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/serializer/AvroToProtoSerializer.java

```diff
@@ -16,12 +16,36 @@
 
 package com.google.cloud.flink.bigquery.sink.serializer;
 
+import com.google.cloud.flink.bigquery.common.utils.SchemaTransform;
 import com.google.cloud.flink.bigquery.sink.exceptions.BigQuerySerializationException;
 import com.google.protobuf.ByteString;
+import org.apache.avro.Schema;
 import org.apache.avro.generic.GenericRecord;
 
 /** Serializer for converting Avro's {@link GenericRecord} to BigQuery proto. */
 public class AvroToProtoSerializer implements BigQueryProtoSerializer<GenericRecord> {
+    /**
+     * Function to convert TableSchema to Avro Schema.
+     *
+     * @param tableSchema A {@link com.google.api.services.bigquery.model.TableSchema} object to
+     *     cast to {@link Schema}
+     * @return Converted Avro Schema
+     */
+    private Schema getAvroSchema(com.google.api.services.bigquery.model.TableSchema tableSchema) {
+        return SchemaTransform.toGenericAvroSchema("root", tableSchema.getFields());
+    }
+
+    /**
+     * Constructor for the Serializer.
+     *
+     * @param tableSchema Table Schema for the Sink Table ({@link
+     *     com.google.api.services.bigquery.model.TableSchema} object )
+     */
+    public AvroToProtoSerializer(com.google.api.services.bigquery.model.TableSchema tableSchema) {
+        Schema avroSchema = getAvroSchema(tableSchema);
+    }
+
+    public AvroToProtoSerializer() {}
```

Comment:
Empty default constructor doesn't need to be explicitly defined.

Reply:
To support the [AvroToProtoSerialiserTest](https://github.com/GoogleCloudDataproc/flink-bigquery-connector/blob/c615f6afb098420356ac00b666ee5b88a5928eb4/flink-connector-bigquery/src/test/java/com/google/cloud/flink/bigquery/sink/serializer/AvroToProtoSerializerTest.java#L32) This is required. Can remove once tests are in.